projects/netgrif-components/schematics/view/create-view-prompt/views/case-view/create-case-view.ts

```diff
@@ -31,8 +31,8 @@ export function createCaseView(tree: Tree, args: CreateViewArguments & TabbedVie
         configName: projectInfo.projectNameClassified,
         configImportPath: createRelativePath(view.fileImportPath, `./${projectInfo.projectNameDasherized}-configuration.service`),
         caseCreationConfig: {
-            enableCaseTitle: args.enableCaseTitle,
-            isCaseTitleRequired: args.isCaseTitleRequired
+            enableCaseTitle: args.enableCaseTitle === undefined ? true : args.enableCaseTitle,
+            isCaseTitleRequired: args.isCaseTitleRequired === undefined ? false : args.isCaseTitleRequired
```

Comment:
In the above two lines consider using nullish coalescing operator, like `args.enableCaseTitle ?? true`.

Reply:
Depends on https://github.com/netgrif/components/pull/94